packages/react-form-state/src/components/Nested.tsx

```diff
@@ -49,7 +54,7 @@ export default class Nested<Fields> extends React.Component<
           name: `${name}.${fieldPath}`,
           initialValue: initialFieldValue,
           dirty: value !== initialFieldValue,
-          error: get(error, fieldPath),
+          error: error[fieldPath],
```

Comment:
I don't understand why we were using `get` here. doesn't mapObject just give us the root keys? cc @TheMallen